scico/test/linop/test_radon_svmbir.py

```diff
@@ -92,3 +112,29 @@ def test_prox_weights(Nx, Ny, num_angles, num_channels, is_3d):
     W = scico.linop.Diagonal(weights)
     f = SVMBIRWeightedSquaredL2Loss(y=sino, A=A, W=W)
     prox_test(v, f, f.prox, alpha=0.25)
+
+
+@pytest.mark.parametrize("Nx, Ny, num_angles, num_channels", (SMALL_INPUT,))
+@pytest.mark.parametrize("is_3d", (True, False))
+@pytest.mark.parametrize("is_weighted", (True, False))
+def test_prox_cg(Nx, Ny, num_angles, num_channels, is_3d, is_weighted):
+    im = make_im(Nx, Ny, is_3d=is_3d) / Nx * 10
+    A = make_A(im, num_angles, num_channels)
+    y = A @ im
+
+    if is_weighted:
+        W = snp.exp(-y) * 20
+    else:
+        W = snp.ones_like(y)
+
```

Comment:
Also support `W=None`?

Reply:
Would it make more sense to use the svmbir.calc_weights() here since that is what the examples use?

Reply:
We could then pass the various strings ("unweighted", "transmission" etc) as the test argument

Reply:
Good idea.

Reply:
Updated the test